examples/demo-app/src/utils/sharing/dropbox.js

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+// DROPBOX
+import {Dropbox} from 'dropbox';
+import {parseQueryString} from '../url';
+
+const DROPBOX_CLIEND_ID = process.env.DropboxClientId;
+const NAME = 'dropbox';
+const DOMAIN = 'www.dropbox.com';
+const CORS_FREE_DOMAIN = 'dl.dropboxusercontent.com';
+const dropbox = new Dropbox({clientId: DROPBOX_CLIEND_ID});
+
+/**
+ * Generate auth link url to open to be used to handle OAuth2
+ * @param {string} path
+ */
+function authLink(path = 'auth') {
+  return dropbox.getAuthenticationUrl(
+    `${window.location.origin}/${path}`,
```

Comment:
you can `import window from 'global/window'`  in which case you can still run this in test mode, and don't need to add to .eslintrc

Reply:
done